app/src/androidTest/java/ch/epfl/skysync/database/tables/MessageGroupTest.kt

```diff
@@ -0,0 +1,100 @@
+package ch.epfl.skysync.database.tables
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import ch.epfl.skysync.database.DatabaseSetup
+import ch.epfl.skysync.database.FirestoreDatabase
+import ch.epfl.skysync.database.ListenerUpdate
+import ch.epfl.skysync.models.message.Message
+import java.time.Instant
+import java.util.Date
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MessageGroupTest {
+  private val db = FirestoreDatabase(useEmulator = true)
+  private val dbs = DatabaseSetup()
+  private val messageGroupTable = MessageGroupTable(db)
+  private val messageTable = MessageTable(db)
+
+  @Before
+  fun testSetup() = runTest {
+    dbs.clearDatabase(db)
+    dbs.fillDatabase(db)
+  }
+
+  @Test
+  fun getTest() = runTest {
+    val messageGroup = messageGroupTable.get(dbs.messageGroup1.id, onError = { assertNull(it) })
+    assertNotNull(messageGroup)
+    assertTrue(
+        listOf(dbs.admin2.id, dbs.pilot1.id, dbs.crew1.id).containsAll(messageGroup!!.userIds))
```

Comment:
This is true even if messageGroup !!.userIds is empty. It would be better if it were the other way around.

Reply:
Good catch, will fix